package.json

```diff
@@ -5,10 +5,15 @@
   "author": "Christian Joudrey",
   "main": "lib/index.js",
   "scripts": {
-    "precommit": "lint-staged",
     "test": "mocha test/index.js",
     "prepare": "rm -rf lib/* && babel ./src --ignore test --out-dir ./lib"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "yarn test"
```

Comment:
Imagine you're working on a change and want to push something that is work in progress so that you have a copy off your local machine. If I understand this correctly, this would prevent anyone from doing that until they fix the tests.

Reply:
yea, One can easily do `--no-verify` to commit and push anyways. but they will know this is not the right state for a PR. 
My motivation is mainly that we don't have Github actions in here yet (I can add them as well if you are interested) and this check will give you a bit more trust in the state of the PR.

Reply:
I'll continue to think about this, but for now I'd prefer not changing my local workflow.